dipy/viz/actor.py

```diff
@@ -0,0 +1,648 @@
+        number of splines subdivision to smooth streamtubes
+    lookup_colormap : bool
+        add a default lookup table to the colormap
+
+    Examples
+    --------
+    >>> from dipy.viz import fvtk
+    >>> r=fvtk.ren()
+    >>> lines=[np.random.rand(10, 3), np.random.rand(20, 3)]
+    >>> colors=np.random.rand(2, 3)
+    >>> c=fvtk.streamtube(lines, colors)
+    >>> fvtk.add(r,c)
+    >>> #fvtk.show(r)
+
+    Notes
+    -----
+    Streamtubes can be heavy on GPU when loading many streamlines and
+    therefore, you may experience slow rendering time depending on system GPU.
+    A solution to this problem is to reduce the number of points in each
+    streamline. In Dipy we provide an algorithm that will reduce the number of
+    points on the straighter parts of the streamline but keep more points on
+    the curvier parts. This can be used in the following way
+
+    from dipy.tracking.distances import approx_polygon_track
+    lines = [approx_polygon_track(line, 0.2) for line in lines]
+
+    Alternatively we suggest using the ``line`` actor which is much more
+    efficient.
+
+    See Also
+    --------
+    dipy.viz.actor.line
+    """
+    # Poly data with lines and colors
+    poly_data, is_colormap = lines_to_vtk_polydata(lines, colors)
+    next_input = poly_data
+
+    # Set Normals
+    poly_normals = set_input(vtk.vtkPolyDataNormals(), next_input)
+    poly_normals.ComputeCellNormalsOn()
+    poly_normals.ComputePointNormalsOn()
+    poly_normals.ConsistencyOn()
+    poly_normals.AutoOrientNormalsOn()
+    poly_normals.Update()
+    next_input = poly_normals.GetOutputPort()
+
+    # Spline interpolation
+    if (spline_subdiv is not None) and (spline_subdiv > 0):
+        spline_filter = set_input(vtk.vtkSplineFilter(), next_input)
+        spline_filter.SetSubdivideToSpecified()
+        spline_filter.SetNumberOfSubdivisions(spline_subdiv)
+        spline_filter.Update()
+        next_input = spline_filter.GetOutputPort()
+
+    # Add thickness to the resulting lines
+    tube_filter = set_input(vtk.vtkTubeFilter(), next_input)
+    tube_filter.SetNumberOfSides(tube_sides)
+    tube_filter.SetRadius(linewidth)
+    # tube_filter.SetVaryRadiusToVaryRadiusByScalar()
+    tube_filter.CappingOn()
+    tube_filter.Update()
+    next_input = tube_filter.GetOutputPort()
+
+    # Poly mapper
+    poly_mapper = set_input(vtk.vtkPolyDataMapper(), next_input)
+    poly_mapper.ScalarVisibilityOn()
+    poly_mapper.SetScalarModeToUsePointFieldData()
+    poly_mapper.SelectColorArray("Colors")
+    poly_mapper.GlobalImmediateModeRenderingOn()
+    # poly_mapper.SetColorModeToMapScalars()
+    poly_mapper.Update()
+
+    # Color Scale with a lookup table
+    if is_colormap:
+        if lookup_colormap is None:
+            lookup_colormap = colormap_lookup_table()
+        poly_mapper.SetLookupTable(lookup_colormap)
+        poly_mapper.UseLookupTableScalarRangeOn()
+        poly_mapper.Update()
+
+    # Set Actor
+    if lod:
```

Comment:
What's the difference between vtkActor and vtkLODActor? We would want to explain this in the docstring, so that people know when to use which